flake.nix

```diff
@@ -0,0 +1,25 @@
+{
+  description = "A plugin to make your hyprland cursor more realistic, also adds shake to find.";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
+    hyprland.url = "git+https://github.com/hyprwm/Hyprland?submodules=1";
+  };
+
+  outputs = {
+    self,
+    nixpkgs,
+    ...
+  } @ inputs: let
+    forAllSystems = function:
+      nixpkgs.lib.genAttrs [
+        "x86_64-linux"
+        "aarch64-linux"
```

Comment:
this plugin makes heavy use of function hooks, which are not supported on arm iirc

Reply:
hm I haven't tested arm either, so I'll just remove that for now